tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "node16",
```

Comment:
Is there any specific reason for `node16` instead of just `node`?

Reply:
When just `node` is used, TypeScript doesn't recognize [exports](https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing) field of `package.json` (it's used by `@fpjs-incubator/broyster`)